frontend/src/text_resources.js

```diff
@@ -330,85 +324,85 @@ let LOCALIZE = new LocalizedStrings({
         }
       },
       emails: {
+        email0: {
+          id: "0",
+          to: "To 0",
+          from: "From 0",
+          subject: "Subject 0",
+          date: "Date 0",
+          body: "Body 0"
+        },
         email1: {
-          id: 1,
+          id: "1",
```

Comment:
is it necessary to translate these since they are numbers?

Reply:
Apparently I can use them as Strings for now. In a later solution, these will be either the id or an orderBy number from the database.